.rubocop.yml

```diff
@@ -1,5 +1,12 @@
 inherit_from: .rubocop_todo.yml
 
+AllCops:
+  TargetRubyVersion: 1.9
+
+# not Ruby 1.9.3 compatible
```

Comment:
Current master fails on travis due to new cops added to rubocop.

Those cops enforce implementations that are not ruby 1.9 compatible (e.g. introduce %i[] for symbols from ruby 2.0)

For details: look at the history of this PR.

First attempt made the code rubocop compatible. It failed on ruby 1.9.3
https://travis-ci.org/commander-rb/commander/builds/267584518

Second attempt was some fiddlings that failed as well: https://travis-ci.org/commander-rb/commander/builds/267584518

Last attempt disabled the cops that are not Ruby 1.9 compatible. This passes the build everywhere. https://travis-ci.org/commander-rb/commander/builds/267591662